config/main.py

```diff
@@ -1271,6 +1282,16 @@ def save(filename):
         with open(file, 'w') as config_db_file:
```

Comment:
>config_db_file [](http://example.com/codeflow?start=32&length=14)

If user want to save to one file by providing expected command line argument, we should not save individual files for each asic? That is one of the reason that audience want to review HLD.

Reply:
We don't save individual files for each asic. For multi-asic, it just save to one file finally.

Reply:
Then why it is in a for-loop?